src/js/components/MultiSelect/stories/ObjectOptions.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+
+import { Box, Text } from 'grommet';
+import { MultiSelect } from '../MultiSelect';
+
+const objectOptions = [
+  {
+    label: 'Red',
+    value: 1,
+  },
+  {
+    label: 'Blue',
+    value: 2,
+  },
+  {
+    label: 'Green',
+    value: 3,
+  },
+  {
+    label: 'Purple',
+    value: 4,
+  },
+];
+
+export const ObjectOptions = () => {
+  const [options, setOptions] = useState(objectOptions);
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" justify="start" pad="large">
```

Comment:
```suggestion
    <Box fill gap='large' align="center" justify="start" pad="large">
```


can we add `gap` to match layout of other stories?

Reply:
Yes, added